stdlib/@python2/bz2.pyi

```diff
@@ -1,9 +1,9 @@
 import io
-from _typeshed import AnyPath, ReadableBuffer, WriteableBuffer
-from typing import IO, Any, Iterable, List, Optional, TypeVar, Union
+from _typeshed import ReadableBuffer, WriteableBuffer
+from typing import IO, Any, Iterable, List, Optional, Text, TypeVar, Union
```

Comment:
Why not just use `unicode` instead of `Text` in Python 2?

Reply:
I think we agreed at some point that semantically `Text` means str or unicode and `unicode` means just unicode.

Reply:
The information about this is all a bit hidden, but `unicode` and `str` are not compatible on all type checkers (e.g. pytype). Therefore the following can fail to type check:

```python
def foo(x: unicode): ...
foo("")
```

But PEP 484 also says:

> For parameters typed as `Text`, arguments of type `str` as well as `unicode` should be acceptable.

So the following checks on all type checkers:

```python
def foo(x: Text): ...
foo("")
```